ludwig/schema/preprocessing.py

```diff
@@ -177,6 +177,7 @@ class NumberPreprocessingConfig(schema_utils.BaseMarshmallowConfig):
 
 @register_preprocessor(BINARY)
 @dataclass
+@register_preprocessor(BINARY)
```

Comment:
Not sure if `@register_preprocessor(BINARY)` is accidentally duplicated twice or supposed to be like this intentionally?

Reply:
Ah yes this was a mistake, just found this in my current branch, will remove.